cli/cmd/upgrade.go

```diff
@@ -161,8 +161,14 @@ The upgrade can be configured by using the --set, --values, --set-string and --s
 A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/tree/main/charts/linkerd2/README.md
 `,
 
-		Example: `  # Default upgrade.
-  linkerd upgrade | kubectl apply --prune -l linkerd.io/control-plane-ns=linkerd -f -
+		Example: `  # Default upgrade - also removes linkerd resources that no longer exist in the current version
+  linkerd upgrade control-plane | kubectl apply -f -
```

Comment:
is this right? I thought that we needed the prune flag on this command in order to prune namespace scoped resources.  I believe that the command below which adds the prune-whitelist will only prune the specified cluster-scoped resources.

Reply:
You're right, copy-pasta error :grimacing: